frontend/package.json

```diff
@@ -6,12 +6,16 @@
     "@testing-library/jest-dom": "^5.16.3",
     "@testing-library/react": "^12.1.4",
     "@testing-library/user-event": "^13.5.0",
+    "i": "^0.3.7",
+    "npm": "^8.10.0",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
     "react-icons": "^4.3.1",
     "react-konva": "^18.0.0-0",
+    "react-material-tabs": "^0.0.8",
```

Comment:
Some packages added in this PR seem to be unused: `i`, `npm` and `react-material-tabs`